llvm/test/DebugInfo/X86/debug-aranges-sce-tuning.ll

```diff
@@ -0,0 +1,33 @@
+; This checks that .debug_aranges is always generated for the SCE debugger
+; tuning.
+
+; RUN: llc -mtriple=x86_64 -debugger-tune=sce -filetype=obj %s -o %t
+; RUN: llvm-dwarfdump -debug-aranges %t | FileCheck %s
+
+; CHECK:      .debug_aranges contents:
+; CHECK-NEXT: Address Range Header:
+; CHECK-SAME:   length = 0x0000002c,
+
+; IR generated and reduced from:
+; $ cat foo.c
+; int foo;
+; $ clang -g -S -emit-llvm foo.c -o foo.ll
+
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
This is redundant with the explicit triple on the llc command line.

Reply:
`clang --target=x86_64-linux-gnu` above and the the target triple in IR might be slightly useful to omit `-mtriple` for llc.

Reply:
I think I've done this - if I got your meaning(?)